pkg/assets/handlers.go

```diff
@@ -154,6 +154,22 @@ type WebConsoleVersion struct {
 	OpenShiftVersion  string
 }
 
+var extensionPropertiesTemplate = template.Must(template.New("webConsoleExtensionProperties").Parse(`
+window.OPENSHIFT_EXTENSION_PROPERTIES = {
+  {{ $values := .ExtensionPropertyValues }}
+  {{ range $i, $key := .ExtensionPropertyKeys }}
+  {{ if $i }},{{ end }}
+  {{ $value := index $values $i }}
+  "{{ $key | js }}": "{{ $value | js }}"
```

Comment:
If you're worried about empty values, how about restricting them in validation for the object?